scripts/hub_bootstrap.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install python3-venv python3-dev python3-setuptools
```

Comment:
It's better to make `apt-get` commands non-interactional with specifying `-y` flag.
e.g. `apt-get -y install`.

Also calling `apt-get upgrade` will update the whole system, make sure that you need that.